### ISSUE TITLE
dotnet-format.exe 3.0.0 is looking for 3.0.0-beta4-19105-10 assemblies

### ISSUE THREAD

**Issue by paulomorgado** [dotnet/roslyn#33414]
**Version Used**: 

dotnet-format.exe 3.0.0

**Steps to Reproduce**:

1. Install dontet-format 3.0.0 - `dotnet tool install --global dotnet-format --version 3.0.0 `
2. run dotnet-format
3. 

**Expected Behavior**:

Runs!

**Actual Behavior**:

```
The application to execute does not exist: 'C:\Users\me\.dotnet\tools\.store\dotnet-format\3.0.0-beta4-19105-10\dotnet-format\3.0.0-beta4-19105-10\tools\netcoreapp2.1\any\dotnet-format.dll'.
```

**Comment by jmarolf** [comment]
I am seeing it placed under 

`C:\Users\me\.dotnet\tools\.store\dotnet-format\3.0.0\dotnet-format\3.0.0\tools\netcoreapp2.1\any\dotnet-format.dll`

but calls to `dotnet-format` expect it to be 

`C:\Users\me\.dotnet\tools\.store\dotnet-format\3.0.0-beta4-19105-10\dotnet-format\3.0.0-beta4-19105-10\tools\netcoreapp2.1\any\dotnet-format.dll`

@livarcocc is this a known bug in the SDK?

**Comment by jmarolf** [comment]
@paulomorgado as a workaround you can run this:

```
dotnet tool install -g dotnet-format --version 3.0.0-beta4-19105-10
```

**Comment by jmarolf** [comment]
@JoeRobich 	and @genlu 	which build is this package (https://www.nuget.org/packages/dotnet-format/3.0.0) based off of?  It appears to not work correctly with the dotnet sdk

**Comment by livarcocc** [comment]
@wli3

**Comment by paulomorgado** [comment]
Commit [70a2f514eb3932648f771b086834cb0a376fee20](https://github.com/dotnet/roslyn/tree/70a2f514eb3932648f771b086834cb0a376fee20), according to the nuspec.

I see that it has been unlisted.

**Comment by jmarolf** [comment]
yeah, we've unlisted it for now so the text on nuget.org doesn't recommend you do something that will run into this problem.

**Comment by KexyBiscuit** [comment]
The problem still exists.

```powershell
> dotnet tool install --global dotnet-format
You can invoke the tool using the following command: dotnet-format
Tool 'dotnet-format' (version '3.0.0') was successfully installed.
> dotnet format
The application to execute does not exist: 'C:\Users\KexyBiscuit\.dotnet\tools\.store\dotnet-format\3.0.0-beta4-19105-10\dotnet-format\3.0.0-beta4-19105-10\tools\netcoreapp2.1\any\dotnet-format.dll'.
```

**Comment by KexyBiscuit** [comment]
Seems that `dotnet tool install --global` doesn't respect NuGet unlisting settings?

**Comment by jmarolf** [comment]
@KexyBiscuit 	nuget doesn't ever delete packages, they are unlisted but they are still there.  Running this will still pull the invalid `3.0.0` package
```commandline
dotnet tool install --global dotnet-format
```
You will need to run this in while I see how to fix this, sorry!
```commandline
dotnet tool install --global dotnet-format --version 3.0.0-beta4-19105-10
```
